main/SS/Formula/PTG/Ptg.cs

```diff
@@ -208,10 +208,10 @@ private static Ptg[] ToPtgArray(ArrayList l)
         //    }
         //    return (Ptg)Clone();
         //}
-        public virtual object Clone()
-        {
-            return this.Copy();
-        }
+        // private virtual object Clone()
+        //{
+            //return this.Copy();
+        //}
```

Comment:
This is not used elsewhere. And `IClonable.Clone` throws `NotImplementedException`, see below.